lib/node.js

```diff
@@ -1006,6 +1006,7 @@ declare module "stream" {
   declare var Writable : typeof stream$Writable
   declare var Duplex : typeof stream$Duplex
   declare var Transform : typeof stream$Duplex
+  declare var PassThrough : typeof stream$Duplex
```

Comment:
it looks like this extends `stream.Transform`, so I think you'd want something like

```
declare class stream$Transform extends stream$Duplex {}
declare class stream$PassThrough extends stream$Transform {}
declare module "stream" {
  ...
  declare var Transform : typeof stream$Transform
  declare var PassThrough : typeof stream$PassThrough
}
```

saying that it's a Duplex seems accurate but not totally precise.